diff --git a/node_modules/expo-firebase-recaptcha/build/FirebaseRecaptchaVerifier.js b/node_modules/expo-firebase-recaptcha/build/FirebaseRecaptchaVerifier.js
index ba7804c..3371ab2 100644
--- a/node_modules/expo-firebase-recaptcha/build/FirebaseRecaptchaVerifier.js
+++ b/node_modules/expo-firebase-recaptcha/build/FirebaseRecaptchaVerifier.js
@@ -9,5 +9,7 @@ export default class FirebaseRecaptchaVerifier {
     async verify() {
         return this.token;
     }
+    async _reset(){
+    }
 }
 //# sourceMappingURL=FirebaseRecaptchaVerifier.js.map
\ No newline at end of file
